contracts/truefi/TrueLender.sol

```diff
@@ -56,29 +41,18 @@ contract TrueLender is Ownable {
     event VotingPeriodChanged(uint256 votingPeriod);
     event SizeLimitsChanged(uint256 minSize, uint256 maxSize);
     event DurationLimitsChanged(uint256 minDuration, uint256 maxDuration);
-    event ApplicationSubmitted(bytes8 id, address borrower, address beneficiary, uint256 amount, uint256 apy, uint256 duration);
-    event ApplicationRetracted(bytes8 id);
-
-    modifier onlyAllowed() {
-        require(borrowers[msg.sender], "TrueLender: sender not allowed borrower");
-        _;
-    }
-
-    modifier applicationExists(bytes8 id) {
-        require(applications[id].creationBlock != 0, "TrueLender: application doesn't exist");
-        _;
-    }
+    event Funded(address indexed loanToken, uint256 amount);
 
-    modifier onlyDuringVoting(bytes8 id) {
-        require(status(id) == ApplicationStatus.Pending, "TrueLender: can't vote outside the voting period");
+    modifier onlyAllowedBorrowers() {
+        require(allowedBorrowers[msg.sender], "TrueLender: sender not allowed borrower");
```

Comment:
`sender not allowed borrower` doesn't sound correct. Not sure how to phrase it. Maybe `sender is not allowed to borrow`?

Reply:
Oh and some errors start from capital letter, some don't. Let's choose one approach